CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.0.3 (2019-04-25)
+
+- Fixed flyout component to not close when infinite scroll "Load more" button is clicked. [#30](https://github.com/blackbaud/skyux-flyout/pull/30)
```

Comment:
This describes the issue we addressed, but not the actual change we made to the flyout. Including the infinite scroll's behavior in the flyout's changelog feels wrong?

Reply:
How's that?

Reply:
What about "Fixed flyout component to remain open when users click infinite scroll "Load more" button." ?

Reply:
@Blackbaud-DeniseCisneros sorry, refresh your screen - I just removed "infinite scroll" all together and made it more generic.